IDEAS/Buildings/Components/ZoneAirModels/WellMixedAir.mo

```diff
@@ -22,7 +22,8 @@ protected
     allowFlowReversal=allowFlowReversal,
     V=Vtot,
     mSenFac=mSenFac,
-    use_C_flow=true)                           annotation (Placement(
+    use_C_flow=true,
+    dynBal(U(nominal=mSenFac*10*Vtot*1.2*1000)))                           annotation (Placement(
```

Comment:
Where does this factor 10 come from?

Reply:
I essentially assume that temperature differences are within a range of 10 K.